plugins/webapi/tools/plugin.go

```diff
@@ -13,6 +13,32 @@ import (
 // PluginName is the name of the web API tools endpoint plugin.
 const PluginName = "WebAPI tools Endpoint"
 
+const (
+	routeDiagnostics = "tools/diagnostic"
+	// API route for message diagnostics
```

Comment:
```suggestion
	// RouteDiagnosticMessages is the API route for message diagnostics
```

Reply:
the following should also start with the name of the exported constant as the dog suggest 🐶

Reply:
dog has been fed